sklearn/decomposition/_pca.py

```diff
@@ -273,6 +274,7 @@ class PCA(_BasePCA):
     ----------
     For n_components == 'mle', this class uses the method of *Minka, T. P.
     "Automatic choice of dimensionality for PCA". In NIPS, pp. 598-604*
+    See https://tminka.github.io/papers/pca/
```

Comment:
Could you create an hyperlink here such that the part from `Minka ... 604` would be clickable on the HTML.